### ISSUE TITLE
Files marked as dirty when opening

### ISSUE THREAD

**Issue by OldStarchy** [excalidraw/excalidraw-vscode#81]
Sometimes when I open a `*.excalidraw` file it immediately gets marked as dirty, If I hit save (without making any change) to clear the mark. The actual changes that get saved vary but I've seen the order of the properties change, and a number of property values change; `version`, `versionNonce`, `updated`, but also `height` and `baseline`.

If I then close all editor windows, and then try to close vscode I get "Do you want to save the changes you made to file.excalidraw" popup.

**Comment by MrPeacockNLB** [comment]
Yes, please fix this as it super anoying when using excalidraw file in a git repo. If one has many files in VScode opening a file it changes immedately but you can not figure out which one has real changes in it.

I'm using *.excalidraw.svg as container for the metadata

BTW: excalidraw is awesome!

**Comment by ciosoriog** [comment]
I love working with excalidraw, and more on vscode, but this is extremely annoying and makes it difficult to keep files synced when trying to work with files that were just opened but not changed clean in the source control repo, introducing artifact changes and causing conflicts.

**Comment by xlsigned** [comment]
same here. what is the status here? will this ever get fixed?

**Comment by pomdtr** [comment]
Hey! I don't really have the bandwidth to maintain the extension (besides small updates). 

This project is behind the excalidraw org, but for now I'm the only maintainer.

Any contribution is welcome!
